tests/mathstat/test_power_spectrum.py

```diff
@@ -27,35 +26,3 @@ def test_values():
     expected = np.linspace(0, 0.5, 6)
     for a, b in zip(f, expected):
         assert a == pytest.approx(b)
-
-
-# test functions used in the oct23 true bq analysis
-# where the pulses did not live in ljh files, instead the pulses needed to be passed
-# explicitly as arguments to the psd and autocorr calculation
-def test_orthognal_to_exponential_filter():
```

Comment:
I don't mind removing the Experimental filter, but I would like to keep a test that shows how to go from pulses and noise records to filters.

Reply:
Good idea. Done--see e1f27b7b

Reply:
and f0008b97, with the test function renamed to have an accurate description.